xml/deployment_prep_aarch64_raspi.xml

```diff
@@ -29,162 +29,266 @@
-   <link xlink:href="https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/msd.md"/>
+   It is also possible to connect an RTC via the GPIO connector,
+   using Hardware Attached on Top (HATs) or other expansion boards.
   </para>
+  <para>
+   Either way, please check whether the respective RTC chipset
+   is supported by &productname;.
+   The connected RTC will need to be described to the operating system
+   via a Device Tree Overlay (<xref linkend="sec-aarch64-deprpi-configtxt"/>).
+   For example, the MyPi base board might use:
+  </para>
+  <screen>dtparam=i2c1=on
+dtoverlay=i2c-rtc,ds1307</screen>
+ </sect2>
 
+ <sect2 xml:id="sec-aarch64-deprpi-img">
+  <title>Deploying an Appliance Image</title>
+  <para>
+   The most common method to deploy an operating system onto &rpi; hardware is
+   to copy a pre-installed system image onto a boot medium, usually a
+   &microsd; card. This is the simplest and easiest method.
+  </para>
+  <para>
+   &suse; provides a pre-configured bootable image of &productname; for &rpi;
+   hardware. This comes with the Btrfs file system, with compression enabled
+   to improve performance and reduce wear on &microsd; media.
+  </para>
+  <para>
+   A &microsd; card with a minimum size of 8&nbsp;GB is recommended. Faster cards will
+   give better system performance. On the first boot, the operating system
+   automatically expands the file system to fill the card. This means that the
+   first boot will be substantially slower than subsequent boots.
+  </para>
+  <para>
+   The process of writing the card image onto &microsd; media is described in the
+   <link xlink:href="&dsc-sles;/html/SLES-rpi-quick/art-rpiquick.html">&rpiquick;</link>.
+  </para>
  </sect2>
- <sect2>
+
+ <sect2 xml:id="sec-aarch64-deprpi-usb">
+  <title>Installation from USB Media</title>
+  <para>
+   Some models of &rpi; allow to boot from USB mass storage devices.
+   This will then allow to deploy &productname; on &rpi; similar to server platforms.
+  </para>
+  <para>
+   Installation can be performed from a removable USB medium, such as a memory stick,
+   onto a &microsd; card in the machine's internal slot, or from a removable
+   USB medium onto another USB medium, such as a USB-connected hard disk.
+  </para>
+  <note>
+   <title>USB Bandwidth Limitations</title>
+   <para>
+    Note that the Ethernet controller on the &rpi; 3 is connected to the device's
+    on-board USB 2.0 bus.
+   </para>
+   <para>
+    Therefore an operating system running from a disk attached via USB
+    must share the total 480&nbsp;Mbps bandwidth of the USB 2.0 controller.
+    This will limit performance, and could significantly impact network performance.
+   </para>
+   <para>
+    This limitation does not apply to the &rpi; 4.
+   </para>
+  </note>
+  <para>
+   Newer models of &rpi; 3 with BCM2837 B0 silicon (silver instead of black chip),
+   including &rpi; 3 Model B+ and Compute Module 3+, allow to boot from USB-connected
```

Comment:
"allow to boot" should be "allow booting"

Reply:
@cjschroder Can you please explain why? I've changed a second instance of "allow to boot" as requested above, but there is also "allow to deploy" - is "allow to" generally disliked, or is it "to boot" that is the culprit?

Reply:
> is "allow to" generally disliked

Well, yes. The word following the transitive verb "allow" is its subject: the noun which is being permitted. So a gerund ("booting," "running") is acceptable, but an infinitive verb is not. We would need the wordier "allows the computer to boot", for example.

Reply:
Many thanks for explaining. Addressed in #564.